troposphere/static/js/components/providers/ProvidersMaster.react.js

```diff
@@ -1,72 +1,19 @@
-define(function (require) {
-  "use strict";
+import React from 'react/addons';
+import Backbone from 'backbone';
+import Router from 'react-router';
+import stores from 'stores';
 
-  var React = require('react/addons'),
-    Router = require('react-router'),
-    RouteHandler = Router.RouteHandler,
-    stores = require('stores');
+var RouteHandler = Router.RouteHandler;
```

Comment:
I *only* mention this because in the changes I've been making regarding ES6 module definitions - I've been changing the *top-level* variables within the module from `var` to `let`. This is not important, given that Babel will apply `"use strict";` to the contents of a module. I just started to sprinkle that in for gradual adoption.